docs/roadmap.md

```diff
@@ -0,0 +1,36 @@
+#  IoT Agent Node.js Library
+
+This product is a FIWARE Generic Enabler. If you would like to learn about the overall Roadmap of FIWARE, please check "Roadmap" on the [FIWARE Catalogue](https://www.fiware.org/developers/catalogue/).
+
+### Introduction
+
+This section elaborates on proposed new features or tasks which are expected to be added to the product in the foreseeable future.  There should be no assumption of a commitment to deliver these features on specific dates or in the order given. The development team will be doing their best to follow the proposed dates and priorities, but please bear in mind that plans to work on a given feature or task may be revised.  All information is provided as general guidelines only,  and this section may be revised to provide newer information at any time.
+
+Disclaimer:
+
+* This section has been last updated in January 2019. Please take into account its 
+  content could be obsolete.
+* Note we develop this software in Agile way, so development plan is continuously 
+  under review. Thus, this roadmap has to be understood as rough plan of features 
+  to be done along time which is fully valid only at the time of writing it. This
+  roadmap has not be understood as a commitment on features and/or dates.
+* Some of the roadmap items may be implemented by external community developers, 
+  out of the scope of GE owners. Thus, the moment in which these features will be
+  finalized cannot be assured.
+  
+### Short term
+
+The following list of features are planned to be addressed in the short term, and incorporated in a release of the product:
+- Removal of warnings and deprecation warnings.
+
+
+### Medium term
```

Comment:
Deprecation of NGSIv1 could be also addressed in the medium term.

Reply:
Ok! However, I have adjusted the wording a bit:

"Removal support for NGSIv1 (which currently is deprecated)"

Being strict NGSIv1 is *already* deprecated, so what is pending is to remove the support.

Reply:
Fixed in 70be5fa77d20393f49e521adf56f12754114b4c5